test/e2e/modal.test.js

```diff
@@ -102,29 +104,29 @@ describe('vl-modal', async () => {
     it('als gebruiker kan ik meteen typen in het input-veld in de modal in Safari', async () => {
```

Comment:
we testen toch (nog) niet in safari?

Reply:
Klopt, maar dan staat de test wel al klaar. Was een specifieke bug in Safari, maar kunnen we nu helaas nog niet automatisch dekken. We coveren hiermee wel al dat dit probleem zich niet plots voordoet in Chrome en Firefox.